test/unit/org/apache/cassandra/tools/nodetool/SetGetColumnIndexSizeTest.java

```diff
@@ -66,7 +66,7 @@ public void testZero()
     @Test
     public void testNegative()
     {
-        assertSetInvalidColumnIndexSize("-7", "Invalid data storage: value must be positive, but was -7", 2);
+        assertSetInvalidColumnIndexSize("-7", "Invalid data storage: value must be positive, but was -7", 1);
```

Comment:
why this change?  Seems unrelated to the scope of this patch?

Reply:
Spoke in Slack, switching from ConfigurationException to IllegalArgumentException causes the rc to be 1 rather than 2

Reply:
see `org.apache.cassandra.tools.NodeTool#execute`